assets/apps/dashboard/src/scss/components/_card.scss

```diff
@@ -58,16 +51,11 @@
 		margin: 15px 0 25px;
 	}
 
-	.card-button {
-		margin-top: 60px;
+  	.card-button-group {
+	  display: grid;
+	  grid-template-columns: min-content min-content;
+	  column-gap: 10px;
```

Comment:
If the wrapper is always there, this can be replaced with flex instead of grid. It should work just fine both with and without the pro version.

Something like this would do the trick:

```
  .card-button-group {
	display: flex;
        margin-top: auto;
	a {
	  margin-left: 10px;

	  &:first-child {
		margin: 0;
	  }
	}
  }
```